lib/rest/basic.js

```diff
@@ -0,0 +1,32 @@
+const { Client } = require('./client');
+
+class BasicClient extends Client {
+  constructor(opts) {
+    // eslint-disable-next-line prefer-object-spread
```

Comment:
I don't know how to fix these. When I follow the recommendations, it fails.

Reply:
I'm not sure why the recommendations were written the way they were. I removed the `assign` calls and used object spread and it seems to be happy.

Reply:
It's because I am a dummy. I did `Object.assign(...foo)`. IOW, I used both.